docs/designs/link-output.md

```diff
@@ -57,20 +60,26 @@ Example:
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "/dotnet/articles/b"
+            "target_url": "/dotnet/articles/b",
+            "source_file_url": "https://github.com/dotnet/articles/a.md",
```

Comment:
if the content is from CRR repo, would `source_file_url` be pointing to that repo?

Reply:
Theoretically yes, it should a URL pointing to the CRR repo. Will add a spec test for this scenario.